bin/bump-version.rb

```diff
@@ -80,37 +36,23 @@ def proposed_changes(version, _new_version)
   puts "☑️  updater/Gemfile.lock updated"
 end
 
-proposed_changes = proposed_changes(version, new_version)
-
-# Update CHANGELOG
-if dry_run
-  puts "Would update CHANGELOG:"
-  puts proposed_changes
-else
-  new_changelog_contents = [
-    "## v#{new_version}, #{Time.now.strftime('%e %B %Y').strip}\n",
-    proposed_changes.join("\n") + "\n",
-    CHANGELOG_CONTENTS
-  ].join("\n")
-
-  File.write(CHANGELOG_PATH, new_changelog_contents)
-  puts "☑️  CHANGELOG.md updated"
-end
-
 unless dry_run
+  puts "Now, create the PR"
```

Comment:
To be clear: I'll put up a PR shortly to automate creation of this PR, but wanted to keep that separate from this PR, which simply switches us to GitHub releases.

Reply:
Ope I missed this.